packages/e2e-tests/src/specs/editor/shopping/schema.json

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
We'll need to pull this in but adding for now to get a test working

Reply:
Happy to help with pulling this in automatically, should be trivial

Reply:
Added a GitHub Actions workflow for this now. That should do the trick.

Reply:
Did you commit to this PR?  If yes think I rebased over it --- sorry.

Reply:
yes... let me try to add it again

Reply:
My fault for pushing to your PR right away, but to prevent overrides form happening again in the future using `--force-with-lease` instead of `--force` is useful. (or merging instead of rebasing & force-pushing)